rosdep/python.yaml

```diff
@@ -1449,6 +1449,10 @@ python-scipy:
     utopic_python3: [python3-scipy]
     vivid: [python-scipy]
     vivid_python3: [python3-scipy]
+python-selenium:
+  ubuntu:
+    pip:
+      packages: [selenium]
```

Comment:
Just so you know, we cannot build binary packages for Ubuntu for packages which depend on pip only things. So if you have a package which depends on this key, we will not be able to release a deb package automatically.